examples/tutorial/x/blog/handlers.go

```diff
@@ -369,33 +369,33 @@ func (h ChangeBlogAuthorsMsgHandler) validate(ctx weave.Context, db weave.KVStor
 		return nil, nil, err
 	}
 	if obj == nil || (obj != nil && obj.Value() == nil) {
-		return nil, nil, ErrBlogNotFound()
+		return nil, nil, errors.ErrNotFound.New("blog")
 	}
 
 	blog := obj.Value().(*Blog)
 	// Check main signer is one of the blog authors
 	if findAuthor(blog.Authors, sender.Address()) == -1 {
-		return nil, nil, ErrUnauthorisedBlogAuthor(sender.Address())
+		return nil, nil, errors.ErrUnauthorized.Newf(unauthorisedBlogAuthor, sender.Address())
 	}
 
 	// Get the author index
 	authorIdx := findAuthor(blog.Authors, changeBlogAuthorsMsg.Author)
 	if changeBlogAuthorsMsg.Add {
 		// When removing an author we must ensure it does not exist already
 		if authorIdx >= 0 {
-			return nil, nil, ErrAuthorAlreadyExist(changeBlogAuthorsMsg.Author)
+			return nil, nil, errors.ErrExists.Newf("author: %X", changeBlogAuthorsMsg.Author)
```

Comment:
This could be a duplicate error, right?

Reply:
yes, let's use this. I was looking for exists error and duplicate didn't come up for some reason.